indico/modules/events/registration/client/js/form/fields/ChoicesSetup.jsx

```diff
@@ -8,11 +8,15 @@
 import {nanoid} from 'nanoid';
 import PropTypes from 'prop-types';
 import React from 'react';
-import {Button} from 'semantic-ui-react';
+import {useSelector} from 'react-redux';
+import {Button, Input} from 'semantic-ui-react';
 
 import {Translate} from 'indico/react/i18n';
 import {SortableWrapper, useSortableItem} from 'indico/react/sortable';
 
+import {getCurrency} from '../../form_setup/selectors';
+import '../../../styles/ChoicesSetup.module.scss';
```

Comment:
I'd put this in the same directory as the JSX file. I only used the styles folder for more global stuff which would be strange to keep next to the JS.